crates/fluvio/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fluvio"
-version = "0.21.1"
+version = "0.21.2"
```

Comment:
21.1 is not release, so no need to bump

Reply:
check crate-version [failed](https://github.com/infinyon/fluvio/actions/runs/6702295994/job/18211081175?pr=3628), I had to change it back to 0.21.2